### PR TITLE
New map Sri Lanka added

### PR DESCRIPTION
**Pull request by Abiseban147**

### CODE REVIEW

File: packages/sri-lanka/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# 🇱🇰 Sri Lanka
```
Comment (VictorCazanave) [inline packages/sri-lanka/README.md]: Could you also create a `package.json` file like the other maps please? I forgot to write it in the [contribution guide](https://github.com/VictorCazanave/svg-maps/blob/master/CONTRIBUTING.md) 🙇

Comment (Abiseban147) [reply packages/sri-lanka/README.md]: I will do it 👍

Comment (Abiseban147) [reply packages/sri-lanka/README.md]: @VictorCazanave i added the package.json file ;-).

File: packages/sri-lanka/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/VictorCazanave/svg-maps/tree/master/packages/sri-lanka"
+  },
+  "keywords": [
+    "svg",
+    "map",
+    "sri-lanka"
+  ],
+  "author": "Victor Cazanave <victor.cazanave@gmail.com>",
```
Comment (VictorCazanave) [inline packages/sri-lanka/package.json]: You can use your name 😃

Comment (Abiseban147) [reply packages/sri-lanka/package.json]: Thank you @VictorCazanave i updated my name as author 😃.

**Comment by VictorCazanave** [comment]
Thanks again!

@all-contributors please add @Abiseban147 for code

**Pull request merged**